src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMStep.java

```diff
@@ -20,36 +15,29 @@
 import hudson.scm.SCM;
 import hudson.util.FormValidation;
 import hudson.util.ListBoxModel;
-import org.apache.commons.lang3.StringUtils;
 import org.jenkinsci.Symbol;
 import org.jenkinsci.plugins.workflow.steps.scm.SCMStep;
 import org.kohsuke.stapler.*;
 import org.kohsuke.stapler.verb.POST;
 
+import javax.annotation.Nonnull;
```

Comment:
I would have thought that checkstyle would be unhappy with you for this. (Turns out our checkstyle config is not correct, I may open a PR separately to fix that). Anyway.. I don't think you need this import, we are `Nonnull` by default

Reply:
Thanks for catching. No idea why I added this....